site/en/docs/privacy-sandbox/topics/overview/index.md

```diff
@@ -0,0 +1,112 @@
+
+The Topics API has three main tasks:
+
+-   Map browser activity to topics of interest. With the current design of the Topics API, topics are inferred from the hostnames of pages the user visits. For example, the topic inferred for a website about aquariums might be [/Pets & Animals/Pets/Fish & Aquaria](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md#:~:text=/Pets%20%26%20Animals/Pets/Fish%20%26%20Aquaria).
+-   Calculate the top topics for a user based on their recent browsing activity.
+-   Provide mechanisms to access topics currently of interest to the user, to help select the appropriate ads.
+
+The Topics API provides human-readable, easily understandable topics, so it's possible to provide meaningful controls to users.
+
+### How topics are curated and selected
+
+Topics are selected from a [taxonomy](https://github.com/jkarlin/topics/blob/main/taxonomy_v1.md), with hierarchical categories such as [/Arts & Entertainment/Music & Audio/Soul & R&B](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md#:~:text=/Arts%20%26%20Entertainment/Music%20%26%20Audio/Soul%20%26%20R%26B) and [/Business & Industrial/Agriculture & Forestry](https://github.com/patcg-individual-drafts/topics/blob/main/taxonomy_v1.md#:~:text=106-,/Business%20%26%20Industrial/Agriculture%20%26%20Forestry,-107). These topics have been curated by Chrome for initial testing, but with the goal that the taxonomy becomes a resource maintained by trusted ecosystem contributors. The taxonomy needs to be small enough that many users' browsers will be associated with each topic. Currently the number of topics is 349, but we expect the final number of topics to be between a few hundred and a few thousand.
+
+To avoid sensitive categories, topics must be public, human-curated, and kept updated. The initial taxonomy proposed for testing by Chrome has been human-curated [to exclude categories generally considered sensitive](/docs/privacy-sandbox/topics/#sensitive-topics), such as ethnicity or sexual orientation.
+
+For 10,000 top sites, the Topics API implementation in Chrome uses a manually curated, publicly available [override list](/docs/privacy-sandbox/topics/#colab:~:text=override_list.pb.gz) to map hostnames to topics. For other sites, the Topics API uses a [machine learning](https://royalsociety.org/topics-policy/projects/machine-learning/what-is-machine-learning-infographic/) model to infer topics from hostnames. 
+
+Chrome's implementation of the Topics API downloads a [TensorFlow Lite](/docs/privacy-sandbox/topics/tensorflow.org/lite/guide) file representing the model, so it can be used locally on the user's device. 
```

Comment:
The TensorFlow Lite link is not correct, I assume from a search and replace. Should be https://www.tensorflow.org/lite/guide